### PR TITLE
Style changes

### PR DESCRIPTION
**Pull request by Samson303**
All style changes from the Figma Courtyard except the Calendar

### CODE REVIEW

File: src/components/Chip/chip.module.scss [hunk 1]
```diff
@@ -21,4 +23,16 @@
...
+
+  &:hover {
+    background: #feeef2;
+    box-shadow: 0px 1px 4px #ffc9d8;
+    border-radius: 60px;
+    //transition: background 0.15s ease 0.125s;
+
+    p {
+      color: #ff205c;
+      //transition: color 0.15s ease 0.125s;
```
Comment (MathisBullinger) [inline src/components/Chip/chip.module.scss]: please remove commented-out code

File: src/screens/Main/mentorcard.module.scss [hunk 2]
```diff
@@ -3,12 +3,16 @@
   padding: 0;
   height: 13rem;
   text-decoration: none;
+  border-radius: 0.5rem;
+  border: none;
+  box-shadow: gray;
```
Comment (MathisBullinger) [inline src/screens/Main/mentorcard.module.scss]: `gray` is an invalid value for the `box-shadow`, which is a shorthand for [multiple properties](https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow).

Either way, if you want to change the shadow color of the cards, you will have to change the shadow in `card.module.scss`. So just remove this line.

File: src/components/Chip/chip.module.scss [hunk 3]
```diff
@@ -1,12 +1,14 @@
 .chip {
   display: flex;
   align-content: center;
-  border: 1px solid var(--cl-text-light);
+  border: 1px solid whitesmoke;
```
Comment (MathisBullinger) [inline src/components/Chip/chip.module.scss]: Please don't use the predefined CSS color names, but the exact color in the design, you can find it in Figma. Preferably formatted as hex code (any other format would also be fine, as long as it is the exact color in the design, we just use hex codes in the rest of the styles).

**Review by MathisBullinger — CHANGES_REQUESTED** [review]
We are currently using `yarn`, not `npm` as package manager. So please remove the `package-lock.json`. You might have to remove the `node_modules` directory and install the packages again with yarn. [Here is](https://classic.yarnpkg.com/en/docs/migrating-from-npm/) a handy overview of all the yarn equivalents to the npm commands.

Please also add `_fragmentTypes.json` to `.gitignore` and remove the file from git tracking with `git rm --cached src/_fragmentTypes.json`. It isn't relevant since it automatically gets generated with every build.

**Review by MathisBullinger — DISMISSED** [review]

**Review by Samson303 — COMMENTED** [review]
Changes Approved by Sam303

**Pull request merged**